packages/libs/react-ui/src/styles/vars.css.ts

```diff
@@ -53,7 +53,7 @@ export const vars = createGlobalTheme(':root', {
   },
   shadows: {
     // TODO: Update to match design system
```

Comment:
Is this todo now complete?

Reply:
No, we're not actually using this in our sprinkles. I'm not sure if we will add shadows with the new token update, but I decided to leave this here for now

Reply:
Alrighty